uat/src/main/java/com.aws.greengrass/FileSteps.java

```diff
@@ -39,8 +39,7 @@ public class FileSteps {
     private final Platform platform;
     private final TestContext testContext;
     private final ScenarioContext scenarioContext;
-
-    private final String ACTIVE_FILE = "ActiveFile";
+    private static final String activeFile = "ActiveFile";
```

Comment:
```suggestion
    private static final String ACTIVE_FILE = "ActiveFile";
```
nit: I agree with the change from `final `to `static final String`. But `ACTIVE_FILE` may be still a good name, since it implies the static final String. By saying that, in the following code, when seeing `this.activeFile`, I thought it was the file object instead of a string.

Reply:
I changes it to that based on the checktyle rules... Let me try `ACTIVEFILE`